### ISSUE TITLE
Thank You! 92mh/s 3080ti

### ISSUE THREAD

**Issue by xrtw** [NebuTech/NBMiner#595]
https://i.gyazo.com/c4662ab5e4d659d4a5da31f5b903844a.png

**Comment by NucleaR-II** [comment]
can you tell us your settings please ?

**Comment by xrtw** [comment]
core 0, memory +1500, power limit 82%

**Comment by NucleaR-II** [comment]
Thank you, on windows or Linux ?

**Comment by charnet3d** [comment]
That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.

**Comment by xrtw** [comment]
> Thank you, on windows or Linux ?

windows, for linux it should be 3000 memory clock i guess

**Comment by BluefireXD** [comment]
please let us know if you also get random MHs Drops to 40MHs or even 20MHs. Seems there is a Bug or LHR Lock which happens randomly

**Comment by ActorMike** [comment]
> That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.

I got a 3080Ti FE from Best Buy and it will OC to 1500, however, I've found that it's less efficient. I'm getting 350 Eff/Watt and 90+MH/s with core 0, memory +1300, power limit 82% Locking the voltage with the Curve edit to 750Mv on Win10 helped stabilize my clocks as well.

The most important setting for me was to change the command line to  -lhr-mode 1 -lhr 74.8 and I'm working my way higher to see where it starts triggering the LHR.

That card in the OP also has low MTmp which indicates the pads have been modified or it has water cooling, etc.

**Comment by BluefireXD** [comment]
Which driver version are you using?

**Comment by ActorMike** [comment]
39.7 NBminer and 496.49 Nvidia.

**Comment by passolarg0f** [comment]
Hey man, thanks for sharing.  Did you change the thermal pads? My vision oc 3080ti is doing 106c. Just one more question, 
this overclock you put in the afterburner is that it? In nbminer's .bat do I need to put some command to unlock some lhr? thanksssss

**Comment by passolarg0f** [comment]
> > That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.
> 
> using 3080 ti vision

Hey man, thanks for sharing. Did you change the thermal pads? My vision oc 3080ti is doing 106c. Just one more question,
this overclock you put in the afterburner is that it? In nbminer's .bat do I need to put some command to unlock some lhr? thanksssss

**Comment by passolarg0f** [comment]
> That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.


I can't make my 3080ti at 70 or 80mhs, only 65. I'm putting 1200cclock and 1150 memory. But I've done other attempts and nothing, do I need to put something else in nbminer's .bat?

**Comment by charnet3d** [comment]
> 
> 
> > That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.
> 
> I got a 3080Ti FE from Best Buy and it will OC to 1500, however, I've found that it's less efficient. I'm getting 350 Eff/Watt and 90+MH/s with core 0, memory +1300, power limit 82% Locking the voltage with the Curve edit to 750Mv on Win10 helped stabilize my clocks as well.
> 
> The most important setting for me was to change the command line to -lhr-mode 1 -lhr 74.8 and I'm working my way higher to see where it starts triggering the LHR.
> 
> That card in the OP also has low MTmp which indicates the pads have been modified or it has water cooling, etc.

Yeah my max lhr value is 74.5. I had to crank the fans quite high to stay below 98c in mtemp (+ a small side exhaust fan just for good measure), I too am waiting for gelid extreme pads and some kryonaut for the repad/repaste.

**Comment by charnet3d** [comment]
> 
> 
> > That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.
> 
> I can't make my 3080ti at 70 or 80mhs, only 65. I'm putting 1200cclock and 1150 memory. But I've done other attempts and nothing, do I need to put something else in nbminer's .bat?

My exact OC is +1102 mem, 1500mhz core fixed through the curve at 755mv (750mv doesn't stick somehow), power limit at 82 just in case.

NBMiner arguments:
-a ethash -o POOL -u WALLET.RIG -lhr 74.5 -lhr-reduce-value 0.1 -log --api 0.0.0.0:22333

You can try looking for the right LHR value for you: Start at something a bit higher like 75 and keep decreasing when it locks in increments of 0.1. The miner does this automatically but I feel like if it's restarted from the beginning each time with the right value it could hold it better.

Also make sure you're using the latest nbminer and nvidia driver.

**Comment by passolarg0f** [comment]
> > > That's a heck of a lottery you got, I can't get my mem above 1300, but I do touch 90 once in a while when my display isn't on.
> > 
> > 
> > I can't make my 3080ti at 70 or 80mhs, only 65. I'm putting 1200cclock and 1150 memory. But I've done other attempts and nothing, do I need to put something else in nbminer's .bat?
> 
> My exact OC is +1102 mem, 1500mhz core fixed through the curve at 755mv (750mv doesn't stick somehow), power limit at 82 just in case.
> 
> NBMiner arguments: -a ethash -o POOL -u WALLET.RIG -lhr 74.5 -lhr-reduce-value 0.1 -log --api 0.0.0.0:22333
> 
> You can try looking for the right LHR value for you: Start at something a bit higher like 75 and keep decreasing when it locks in increments of 0.1. The miner does this automatically but I feel like if it's restarted from the beginning each time with the right value it could hold it better.
> 
> Also make sure you're using the latest nbminer and nvidia driver.

Thankss. I think my card is having problems at Eth because of the high temperatures. always close to 105-110c and with thermal throtling. I've tested several parameters and I can't get the hash above 65mh / s. I'm thinking about activating the Gigabyte warranty without saying anything about mining, do you think they can change the vga or do something? I've already thought about changing the thermal pads, but as I've never done it, I can do something wrong or even not solve the problem, that's why I'm afraid.

**Comment by charnet3d** [comment]
try setting the fans higher, I set my fan curve to ramp up quite high when core is >50c to avoid reaching 104-106 mem and higher. Right now my fans are at 93% and the mem doesn't go above 98c or just touches 100 and goes back down.

**Comment by ActorMike** [comment]
For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393

Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.

**Comment by afhacker** [comment]
I have three Gigabyte 3080 TI Eagle and they are really bad cards for mining, if you ever come around them don't buy them!
Not a single miner LHR unlock works on these cards, and the hashrate for Ravencoin and Ergo is even much lower in comparison to my other 3080 TI cards, right now my best 3080 TI card is ASUS TUF gaming.

For ravencoin they can't go above 40 mh/s, my ASUS cards mine raven at 55 mh/s.

If anyone has these cards and were able to go above 70 mh/s on ETHASH please let me know how you did it.

**Comment by NucleaR-II** [comment]
I also have the problem with Gigabyte 3080 TI Eagle, they do not go above 66Mh/s. I do not want to change the pads because then I lose the warranty. I would be interested to know if it is the pads or if the cards have a new LHR lock.

**Comment by charnet3d** [comment]
> 
> 
> For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> 
> Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.

Even when there's no risk of damage, the throttling can still happen, especially when some vendors set the limit quite low like 104c. The guy in the video himself suggested to try to keep it around 95 or even 90 to be safe in case ambient temp varies. The pad change gives the peace of mind that the card is definitely safe and no perf is lost.
In my situation, I just can't close the side panel with the card mining because I start creeping into 106c territory and the core freq starts to dip a bit, and I have the big NZXT phantom 630 case with more than enough fans. If I want to avoid dust and lower some noise I'll need those pads unfortunately.


> I have three Gigabyte 3080 TI Eagle and they are really bad cards for mining, if you ever come around them don't buy them!
> Not a single miner LHR unlock works on these cards, and the hashrate for Ravencoin and Ergo is even much lower in comparison to my other 3080 TI cards, right now my best 3080 TI card is ASUS TUF gaming.

Yay, I'm lucky then the TUF is my one and only 3080ti :3

You know about the gigabyte cards you can give bios change a try, but that's certainly quite risky if something weird happens (or you don't know what you're doing).
I was successful in unlocking the power further for my 3070 ventus 2x with its crappy power limit at 220w (which is max 100% TDP), using an MSI SUPRIM X bios, which has a base TDP of 285w and can even reach 300w. My games have been thanking me, keeping stable clocks without throttling, though the temps did get toasty nearing 80c.
In my experience flashing bioses for my old 1070 and the 3070, the worst that happened was flashing a bad bios and losing video output for the card after the restart, but booting with a 2nd card in the 2nd pcie slot enables you to reflash the other back to a safe bios, on the other side JayzTwoCents bricked a card when flashing a B revision bios into an A revision card, so shit really can go south. I suggest to research very well the subject if you plan on doing it.

**Comment by afhacker** [comment]
> > For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> > Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.
> 
> Even when there's no risk of damage, the throttling can still happen, especially when some vendors set the limit quite low like 104c. The guy in the video himself suggested to try to keep it around 95 or even 90 to be safe in case ambient temp varies. The pad change gives the peace of mind that the card is definitely safe and no perf is lost. In my situation, I just can't close the side panel with the card mining because I start creeping into 106c territory and the core freq starts to dip a bit, and I have the big NZXT phantom 630 case with more than enough fans. If I want to avoid dust and lower some noise I'll need those pads unfortunately.
> 
> > I have three Gigabyte 3080 TI Eagle and they are really bad cards for mining, if you ever come around them don't buy them!
> > Not a single miner LHR unlock works on these cards, and the hashrate for Ravencoin and Ergo is even much lower in comparison to my other 3080 TI cards, right now my best 3080 TI card is ASUS TUF gaming.
> 
> Yay, I'm lucky then the TUF is my one and only 3080ti :3
> 
> You know about the gigabyte cards you can give bios change a try, but that's certainly quite risky if something weird happens (or you don't know what you're doing). I was successful in unlocking the power further for my 3070 ventus 2x with its crappy power limit at 220w (which is max 100% TDP), using an MSI SUPRIM X bios, which has a base TDP of 285w and can even reach 300w. My games have been thanking me, keeping stable clocks without throttling, though the temps did get toasty nearing 80c. In my experience flashing bioses for my old 1070 and the 3070, the worst that happened was flashing a bad bios and losing video output for the card after the restart, but booting with a 2nd card in the 2nd pcie slot enables you to reflash the other back to a safe bios, on the other side JayzTwoCents bricked a card when flashing a B revision bios into an A revision card, so shit really can go south. I suggest to research very well the subject if you plan on doing it.

I did flashed new Bios from Gigabyte, they released a new Bios for the card and it was for fixing the display port bug, I flashed it but nothing changed.


> I also have the problem with Gigabyte 3080 TI Eagle, they do not go above 66Mh/s ( ETH ). I do not want to change the pads because then I lose the warranty. I would be interested to know if it is the pads or if the cards have a new LHR lock.

Its most probably a new LHR, because my cards run very cool, all of them are on my mining rig, the core temp is most of the time below 53 degree.

I tired on both Windows and HiveOS, no difference at all.

I tired LolMiner, GMiner, and NBMiner, different parameters, different OC. nothing works on these cards.

What makes me mad is that these cards can't even mine Ravencoin efficiently, you can't go above 41 mh/s on raven.

I bought the cards from different resellers, and from different countries.

Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!

The only algo that they can are good at is Ergo Autolykos 256 mh/s power limit 215 watt.

**Comment by charnet3d** [comment]
> 
> I did flashed new Bios from Gigabyte, they released a new Bios for the card and it was for fixing the display port bug, I flashed it but nothing changed.
> 

I meant flashing the bios of another known good card, at least another model from the same vendor (you can flash from other vendors as long as it's a 3080ti but doesn't always work)

The bios files can downloaded from here https://www.techpowerup.com/vgabios/?model=RTX+3080+Ti

For example you can flash the VISION OC bios into the EAGLE, and there may not be the same limitation in mining. But I still reiterate that this operation is at your own risk and can be hard to recover from in case of some serious issue.

**Comment by ActorMike** [comment]
> I bought the cards from different resellers, and from different countries.
> 
> Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
Have you considered that the cards are counterfeit or perhaps the components inside are not actually 3080 TI components? This is a very popular product internationally. They just had a heist of a truck here! https://www.ign.com/articles/rtx-30-series-graphics-cards-stolen-truck-heist

**Comment by afhacker** [comment]
> > I bought the cards from different resellers, and from different countries.
> > Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
> > Have you considered that the cards are counterfeit or perhaps the components inside are not actually 3080 TI components? This is a very popular product internationally. They just had a heist of a truck here! https://www.ign.com/articles/rtx-30-series-graphics-cards-stolen-truck-heist

No, its a real 3080 TI, it works great if you game 4k on it! it just isn't good for mining.
When you start a miner with LHR unlock it starts at 100 or even some timers above 100 mh/s, then it fell to below 50 mh/s, without lhr unlock it can mine consistently at around 66 mh/s.

**Comment by ActorMike** [comment]
> When you start a miner with LHR unlock it starts at 100 or even some timers above 100 mh/s, then it fell to below 50 mh/s, without lhr unlock it can mine consistently at around 66 mh/s.
Are you reading the messages? It sounds like it is triggering the LHR. My card NEVER starts at 100. What is the command line you are using to call the miner? Did you read the suggestions above? This is likely your problem.

**Comment by ActorMike** [comment]
> > > I bought the cards from different resellers, and from different countries.
> > > Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
> > > Have you considered that the cards are counterfeit or perhaps the components inside are not actually 3080 TI components? This is a very popular product internationally. They just had a heist of a truck here! https://www.ign.com/articles/rtx-30-series-graphics-cards-stolen-truck-heist
> 
> No, its a real 3080 TI, it works great if you game 4k on it! it just isn't good for mining. When you start a miner with LHR unlock it starts at 100 or even some timers above 100 mh/s, then it fell to below 50 mh/s, without lhr unlock it can mine consistently at around 66 mh/s.

Are you reading the messages? It sounds like it is triggering the LHR. My card NEVER starts at 100. What is the command line you are using to call the miner? Did you read the suggestions above? This is likely your problem.

**Comment by afhacker** [comment]
> > > > I bought the cards from different resellers, and from different countries.
> > > > Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
> > > > Have you considered that the cards are counterfeit or perhaps the components inside are not actually 3080 TI components? This is a very popular product internationally. They just had a heist of a truck here! https://www.ign.com/articles/rtx-30-series-graphics-cards-stolen-truck-heist
> > 
> > 
> > No, its a real 3080 TI, it works great if you game 4k on it! it just isn't good for mining. When you start a miner with LHR unlock it starts at 100 or even some timers above 100 mh/s, then it fell to below 50 mh/s, without lhr unlock it can mine consistently at around 66 mh/s.
> 
> Are you reading the message status of the minor? Is it saying lock detected? It sounds like it is triggering the LHR. My card NEVER starts at 100. What is the command line you are using to call the miner? Did you read the suggestions above? This is likely your problem.

I use HiveOS shell miner terminal, it start with a very good hashrate after the first unlock and then it fells to below 50 mh/s, after that the miner doesn't unlock the card anymore, and it keeps mining at 40-50 mh/s, so the lhr unlock decrease my hash rate instead of increasing it.
If I disable lhr unlock it mines at 66 mh/s without any problem and power fluctuation.

**Comment by passolarg0f** [comment]
> try setting the fans higher, I set my fan curve to ramp up quite high when core is >50c to avoid reaching 104-106 mem and higher. Right now my fans are at 93% and the mem doesn't go above 98c or just touches 100 and goes back down. Either way u have to avoid 110c because that's when it throttles, or it may happen even sooner on some cards.
> 
> But if you can check with warranty and no way to repad the memory then maybe do that, hopefully you get a better card.

True my friend, at this moment the board has a 100% fan and is mining ERG, but with a temperature of 100c-105c, the ETH is not able to raise the hash at all, it only stays at 65-70 and higher temperature, giving thermal throtling of Sometimes. I'm trying to contact the store I made the purchase from. It's still within the 90-day warranty, the problem is that the store from what I've seen on the site doesn't have more plates like the 3080ti, only inferior ones. I will evaluate the possibility of them refunding me the value, but I find it difficult for the time being. I don't know if they will want to do this for the time that has passed.

**Comment by passolarg0f** [comment]
> For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> 
> Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.
My problem is that my GPU doesn't even allow to hash up the ETH, the temperatures go up to 105, 110 quickly and the hash is between 65-70 with various forms of OC I've tested. Also, it gives thermal throtling and the screen gets dark from time to time.

Whoever made the video posted the following comment on their post:

Either way here's a summary -
-> 95-105 degrees celsius - might have problems with your games/software, not a good idea to run it this hot.
-> 120 degrees celsius - memory will fail, shorten lifespan, die (but your GPU probably won't let you throttle past 104-110 degrees celsius anyway)

**Comment by passolarg0f** [comment]
> I have three Gigabyte 3080 TI Eagle and they are really bad cards for mining, if you ever come around them don't buy them! Not a single miner LHR unlock works on these cards, and the hashrate for Ravencoin and Ergo is even much lower in comparison to my other 3080 TI cards, right now my best 3080 TI card is ASUS TUF gaming.
> 
> For ravencoin they can't go above 40 mh/s, my ASUS cards mine raven at 55 mh/s.
> 
> If anyone has these cards and were able to go above 70 mh/s on ETHASH please let me know how you did it.

It's true my friend. My GPU is a VISION OC 3080ti, I've tried 1200, 1300 cclock fixed or curved with 750, 725 volts. Then increase the mclock to 600, 800, 1150, 1200, 1300, 1500 and nothing happens. It simply stays at 65-70mh/s, very high temperatures. Tested nbminer 39.5, .6, .7, minerstat and nothing.

**Comment by passolarg0f** [comment]
> > > For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> > > Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.
> > 
> > 
> > Even when there's no risk of damage, the throttling can still happen, especially when some vendors set the limit quite low like 104c. The guy in the video himself suggested to try to keep it around 95 or even 90 to be safe in case ambient temp varies. The pad change gives the peace of mind that the card is definitely safe and no perf is lost. In my situation, I just can't close the side panel with the card mining because I start creeping into 106c territory and the core freq starts to dip a bit, and I have the big NZXT phantom 630 case with more than enough fans. If I want to avoid dust and lower some noise I'll need those pads unfortunately.
> > > I have three Gigabyte 3080 TI Eagle and they are really bad cards for mining, if you ever come around them don't buy them!
> > > Not a single miner LHR unlock works on these cards, and the hashrate for Ravencoin and Ergo is even much lower in comparison to my other 3080 TI cards, right now my best 3080 TI card is ASUS TUF gaming.
> > 
> > 
> > Yay, I'm lucky then the TUF is my one and only 3080ti :3
> > You know about the gigabyte cards you can give bios change a try, but that's certainly quite risky if something weird happens (or you don't know what you're doing). I was successful in unlocking the power further for my 3070 ventus 2x with its crappy power limit at 220w (which is max 100% TDP), using an MSI SUPRIM X bios, which has a base TDP of 285w and can even reach 300w. My games have been thanking me, keeping stable clocks without throttling, though the temps did get toasty nearing 80c. In my experience flashing bioses for my old 1070 and the 3070, the worst that happened was flashing a bad bios and losing video output for the card after the restart, but booting with a 2nd card in the 2nd pcie slot enables you to reflash the other back to a safe bios, on the other side JayzTwoCents bricked a card when flashing a B revision bios into an A revision card, so shit really can go south. I suggest to research very well the subject if you plan on doing it.
> 
> I did flashed new Bios from Gigabyte, they released a new Bios for the card and it was for fixing the display port bug, I flashed it but nothing changed.
> 
> > I also have the problem with Gigabyte 3080 TI Eagle, they do not go above 66Mh/s ( ETH ). I do not want to change the pads because then I lose the warranty. I would be interested to know if it is the pads or if the cards have a new LHR lock.
> 
> Its most probably a new LHR, because my cards run very cool, all of them are on my mining rig, the core temp is most of the time below 53 degree.
> 
> I tired on both Windows and HiveOS, no difference at all.
> 
> I tired LolMiner, GMiner, and NBMiner, different parameters, different OC. nothing works on these cards.
> 
> What makes me mad is that these cards can't even mine Ravencoin efficiently, you can't go above 41 mh/s on raven.
> 
> I bought the cards from different resellers, and from different countries.
> 
> Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
> 
> The only algo that they are good at is Ergo Autolykos 256 mh/s power limit 215 watt.

You said everything my friend. I've already thought about changing the thermal pads for VISION too, but I don't understand much and I'm afraid of damaging it, apart from losing the warranty; Would you at least know the pad sizes/measures of the 3080tI?

**Comment by passolarg0f** [comment]
> > > > > I bought the cards from different resellers, and from different countries.
> > > > > Hopefully someone or some miners will find a solution for them, until then stay away from Gigabyte 3080 TI eagle!
> > > > > Have you considered that the cards are counterfeit or perhaps the components inside are not actually 3080 TI components? This is a very popular product internationally. They just had a heist of a truck here! https://www.ign.com/articles/rtx-30-series-graphics-cards-stolen-truck-heist
> > > 
> > > 
> > > No, its a real 3080 TI, it works great if you game 4k on it! it just isn't good for mining. When you start a miner with LHR unlock it starts at 100 or even some timers above 100 mh/s, then it fell to below 50 mh/s, without lhr unlock it can mine consistently at around 66 mh/s.
> > 
> > 
> > Are you reading the message status of the minor? Is it saying lock detected? It sounds like it is triggering the LHR. My card NEVER starts at 100. What is the command line you are using to call the miner? Did you read the suggestions above? This is likely your problem.
> 
> I use HiveOS shell miner terminal, it start with a very good hashrate after the first unlock and then it fells to below 50 mh/s, after that the miner doesn't unlock the card anymore, and it keeps mining at 40-50 mh/s, so the lhr unlock decrease my hash rate instead of increasing it. If I disable lhr unlock it mines at 66 mh/s without any problem and power fluctuation.


My 3080ti the same thing happens. I've already tried to set lhr mode 1, 2 and nothing.

**Comment by ActorMike** [comment]
> > For those curious- Micron has stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> > Lots of people are buying new pads to go down to 85-90c and it's unnecessary. The chips are designed to run hot.
> > My problem is that my GPU doesn't even allow to hash up the ETH, the temperatures go up to 105, 110 quickly and the hash is between 65-70 with various forms of OC I've tested. Also, it gives thermal throtling and the screen gets dark from time to time.
> 
> Whoever made the video posted the following comment on their post:
> 
> Either way here's a summary - -> 95-105 degrees celsius - might have problems with your games/software, not a good idea to run it this hot. -> 120 degrees celsius - memory will fail, shorten lifespan, die (but your GPU probably won't let you throttle past 104-110 degrees celsius anyway)

That's not true. Anything under 105 is fine on these ram modules according to Micron- Micron has also stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393

**Comment by passolarg0f** [comment]
> That's not true. Anything under 105 is fine on these ram modules according to Micron- Micron has also stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> 
> Everyone is just freaking out because this is new information and in the past we just had GPU core temperatures which are much lower.

I understand, who am I to say otherwise because I'm not an expert haha.

What's pissing me off isn't even the temperature mainly, but the hash doesn't go above 65-70mh/s. It's like you told the other colleague, it looks like it's an LHR block. I thought it might have something to do with the temperature, because of the thermal throttling and the black screen.

If not the temperature I have no idea what it is.

**Comment by charnet3d** [comment]
> Would you at least know the pad sizes/measures of the 3080tI?

3080ti only has chips on the core side and they're all 2mm. I heard gelid extreme are safer to apply because they're softer and will be squished even when slightly thicker than necessary. So that's what I ordered for myself.
One sheet of 80x40 is enough but you need not get your measurement wrong, otherwise get 2 sheets to have a backup.

You should check on youtube there are more than enough videos on this that you can choose from, but you won't find too many 3080tis, mostly 3080 and 3090 because they had the same issue first, though if you find a similar model it's gonna be the same steps.

**Comment by ActorMike** [comment]
> > That's not true. Anything under 105 is fine on these ram modules according to Micron- Micron has also stated in written documentation that damage does not occur until over 125c https://youtu.be/ph4_Sr8YNXI?t=393
> > Everyone is just freaking out because this is new information and in the past we just had GPU core temperatures which are much lower.
> 
> I understand, who am I to say otherwise because I'm not an expert haha.
> 
> What's pissing me off isn't even the temperature mainly, but the hash doesn't go above 65-70mh/s. It's like you told the other colleague, it looks like it's an LHR block. I thought it might have something to do with the temperature, because of the thermal throttling and the black screen.
> 
> If not the temperature I have no idea what it is.

It's not the temperature. This clearly shoots down that theory:

![temps](https://user-images.githubusercontent.com/267615/140582179-68ef3afb-d2c4-48fc-bd3f-20c6f18e1d7f.jpg)

**Comment by afhacker** [comment]
I was able to fix the low hash rate issue by replacing the thermal pads, now it gives me the same hash rate like my other 3080 TI cards.

I followed this guide: [https://www.reddit.com/r/gpumining/comments/on0xm9/gigabyte_rtx_3080ti_eagle_thermal_pad_replacement/](https://www.reddit.com/r/gpumining/comments/on0xm9/gigabyte_rtx_3080ti_eagle_thermal_pad_replacement/)

Don't buy these cards!!! they sell $2000+ card and they can't use a good thermal pad.

**Comment by weiyaowu-md** [comment]
> > Thank you, on windows or Linux ?
> 
> windows, for linux it should be 3000 memory clock i guess

hello, could you give me a sample running script for linux platform, I don't know how to set core frequency, power and memory frequency

**Comment by ActorMike** [comment]
New Video that shows in detail how to get 90MHs + on the 3080TI FE card on the latest 40.1 version. The version of the Nvidia drivers is crucial!

https://www.youtube.com/watch?v=eOdlK0aeGMQ

**Comment by samurai-no-shogun** [comment]
![image](https://user-images.githubusercontent.com/105001152/167305307-fc6190a6-ae90-4b13-8dd4-84fce375a74c.png)
still having an issue, can anyone help me figure out how to admin allow

**Comment by animaea** [comment]
> ![image](https://user-images.githubusercontent.com/105001152/167305307-fc6190a6-ae90-4b13-8dd4-84fce375a74c.png) still having an issue, can anyone help me figure out how to admin allow

you should run your terminal software as administrator. right click and select "Run as Administrator"

![image](https://user-images.githubusercontent.com/13407536/180440481-5ec43b18-a082-4ccc-b23f-02e0546bd12e.png)
